### ISSUE TITLE
How to get the reported results on SVHN in the paper.

### ISSUE THREAD

**Issue by biuyq** [EricArazo/PseudoLabeling#4]
Hi, 
    Thank you for sharing your excellent work in SSL. I ran your code for CIFAR-10 and CIFAR-100, and they all perform good, and even better than the reported ones. 
    While I changed some necessary parts of CIFAR-10 to set an experiment on SVHN, including the dataloading, re set the mean std to 0.5, and eliminate the flip augmentation. However, the result seems far behind the reported one (around 90% accuracy). Can you please give me some suggestions or just share the source code for reproducing SVHN results? 
   Thanks a lot!

**Comment by EricArazo** [comment]
Hello,
There are very few modifications in the experiments in SVHN. The most important one is the length of the warm-up phase, in this case, we pretrain the model in the labeled samples for 150 epochs (reducing the learning rate in the epochs 50 and 100). Apart from that, there is the change you mentioned in the transformations regarding the flips and the mean and standard deviation to normalize the images. I share the values we used:

mean = [x / 255 for x in [127.5, 127.5, 127.5]]
std = [x / 255 for x in [127.5, 127.5, 127.5]] 

Let me know for any other questions you might have.

**Comment by biuyq** [comment]
> Hello,
> There are very few modifications in the experiments in SVHN. The most important one is the length of the warm-up phase, in this case, we pretrain the model in the labeled samples for 150 epochs (reducing the learning rate in the epochs 50 and 100). Apart from that, there is the change you mentioned in the transformations regarding the flips and the mean and standard deviation to normalize the images. I share the values we used:
> 
> mean = [x / 255 for x in [127.5, 127.5, 127.5]]
> std = [x / 255 for x in [127.5, 127.5, 127.5]]
> 
> Let me know for any other questions you might have.

OK ~ THANKS A LOT, I WILL TRY THIS~
